stdlib/public/core/StringStorage.swift

```diff
@@ -181,12 +181,14 @@ fileprivate func _allocate<T: AnyObject>(
 ) -> (T, realNumTailBytes: Int) {
   _internalInvariant(getSwiftClassInstanceExtents(T.self).1 == numHeaderBytes)
 
+  func roundUp(_ x: Int) -> Int { (x+15) & ~15 }
```

Comment:
Nit-pick: space around the `+`.

Reply:
why?

Reply:
Isn't the general style in the rest of the standard library to have spaces around the operation?

Reply:
Not that I know of, at least not for `*` or when explicitly parenthesized. It is strongly recommended not to omit spaces in a way that appears to violate precedence, of course. I can add them in since this is `+`.